packages/bautajs-core/src/bauta.ts

```diff
@@ -274,8 +275,10 @@ export class BautaJS implements BautaJSInstance {
         'The given instance is not bootstrapped, thus operation schema will be no inherited.'
       );
     }
+
```

Comment:
@Xavier-Redondo please do not commit the changes on this file.

Reply:
done